### ISSUE TITLE
Tab between column name and column type

### ISSUE THREAD

**Issue by lapter57** [xnuinside/simple-ddl-parser#53]
If the column name is separated from the column type only by a tab, then return `[]`
```sql
CREATE TABLE IF NOT EXISTS schema.table
(
    field_1\tSTRING,
    field_2\tTIMESTAMP
);
```

If the first column name is separated from the column type by a tab and some spaces (before or after the tab), but second column name is separated from the `DECIMAL` type only by a tab then an error is returned
```sql
CREATE TABLE IF NOT EXISTS schema.table
(
    field_1\t STRING,
    field_2\tDECIMAL(38,10)
);
```

If I replace all tabs with a space with method `replace('\t', ' ')`, then I get the correct result

**Comment by xnuinside** [comment]
@lapter57 , hi! thanks for the report. I will try to fix it today and release patch version.

**Comment by xnuinside** [comment]
@lapter57 added test https://github.com/xnuinside/simple-ddl-parser/pull/54 and released 0.19.1 with fix. if will be any other issues - feel free to open the new issue report. Thanks one more tim!
